tools/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-link_libraries(simdjson simdjson-internal-flags simdjson-windows-headers)
+link_libraries(simdjson simdjson-internal-flags simdjson-windows-headers cxxopts)
```

Comment:
So here the issue is that, as things currently stand, cxxopts is only available if you have git. We are not assuming (generally) that the user has git... only specific components assume git. That is, it is possible to build simdjson without having git installed.